src/entries/popup/pages/messages/WatchAsset/index.tsx

```diff
@@ -165,7 +164,7 @@ export const WatchAsset = ({
         });
       }
 
-      approveRequest(null);
+      approveRequest(true);
```

Comment:
looking at `ApproveAppRequest.tsx` seems like `approveRequest` and `rejectRequest` send the same msg to background but if the payload is `null` it considers a reject, shouldn't we solve it in that component? 
I think `AddEthereumChain.tsx` also calls `approveRequest(null)`

Reply:
yeah that's exactly the case, so we're sending true instead

maybe your AddEthereumChain is outdated https://github.com/rainbow-me/browser-extension/blob/master/src/entries/popup/pages/messages/AddEthereumChain/index.tsx#L79